api/v1alpha1/oadp_types.go

```diff
@@ -269,7 +296,7 @@ type VolumeOptions struct {
 	// the data mover cache volume
 	//+optional
 	CacheStorageClassName string `json:"cacheStorageClassName,omitempty"`
-	// cacheCapacity determines the size of the restic metadata cache volume
+	// cacheCapacity determines the size of the nodeAgent metadata cache volume
```

Comment:
NIT: Please remove this change, the `VolumeOptions` are advanced options for Volsync datamover's restic, `nodeAgent` is not relevant here.

Reply:
Reverted that line change.